src/lib/tuya.ts

```diff
@@ -788,6 +788,9 @@ const tuyaTz = {
             'motor_speed', 'timer', 'reset_frost_lock', 'schedule_periodic', 'schedule_weekday', 'backlight_mode', 'calibration', 'motor_steering',
             'mode', 'lower', 'upper', 'delay', 'reverse', 'touch', 'program',
             ...[1, 2, 3, 4, 5, 6].map((no) => `schedule_slot_${no}`), 'minimum_range', 'maximum_range', 'detection_delay', 'fading_time',
+            ...[1, 2, 3, 4, 5, 6].map((no) => `power_on_behavior_l${no}`),
```

Comment:
I don't expect this is needed (because `multiEndpoint: true` is set for this device), so adding just `mode` and `light_mode` here should be enough.